app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -41,7 +41,7 @@
 import cn.pedant.SweetAlert.SweetAlertDialog;
 
 /**
- * The main activity class that shows a list of all people with their public keys
+ * The main activity class that shows a list of all contacts and a button to add new people
```

Comment:
This description is a bit misleading imo, the button allows youto obtain new contact from other users